modules/react/pagination/lib/Pagination/AdditionalDetails.tsx

```diff
@@ -1,14 +1,19 @@
 import * as React from 'react';
 import {type, space, typeColors} from '@workday/canvas-kit-react/tokens';
-import {accessibleHide, styled, StyledType} from '@workday/canvas-kit-react/common';
+import {
```

Comment:
I have to ask, can the Pagination refactor go into a separate PR? This is too much in the docs PR. I know other components I reviewed had similar refactors but this one is the biggest. It's 1000 lines not including the MDX changes.

Reply:
I can, but then Pagination docs are inconsistent with all other components. Instead of having a "Style Component" section describing which layout component is used, it lists out hundreds of layout props.